proc_Hahn_echoph.py

```diff
@@ -23,17 +23,15 @@
     s.setaxis(t2-rough_center)
     logger.info(strm(ndshape(s)))
     #}}}
-    #{{{finds best shift according to hermitian function test
-    best_shift = hermitian_function_test(s)
```

Comment:
needed for a few lines down when we shift in frequency domain

Reply:
no, this is redundant

Reply:
needed for `s *= exp(1j*2*pi*best_shift*s.fromaxis('t2'))`

Reply:
there are two hermitian function test in the code

Reply:
I am confused as I only see it used this once...

Reply:
yes, b/c this line is removed.  were it not removed, there would be two

Reply:
(bad wording on my part, I think -- "there **were** two hermitian..."